cmd/harmony.go

```diff
@@ -294,16 +237,8 @@ func main() {
 	consensus.OnConsensusDone = currentNode.PostConsensusProcessing
 	currentNode.State = node.NodeWaitToJoin
 
-	if !*libp2pPD {
-		if consensus.IsLeader {
-			currentNode.State = node.NodeLeader
```

Comment:
do we still need this? may you elaborate?

Reply:
address it and check in the code asap

Reply:
libp2ppd is just a flag to enable libp2p based peer discovery and gossiping. Since we enable it by default, there is no need to have a flag anymore.

Reply:
node.State is not related to libp2p